src/test/javascript/spec/component/tutorial-groups/crud/edit-tutorial-group.component.spec.ts

```diff
@@ -0,0 +1,151 @@
+        TestBed.configureTestingModule({
+            imports: [],
+            declarations: [EditTutorialGroupComponent, LoadingIndicatorContainerStubComponent, TutorialGroupFormStubComponent, MockPipe(ArtemisTranslatePipe)],
+            providers: [
+                MockProvider(TutorialGroupsService),
+                MockProvider(AlertService),
+                { provide: Router, useClass: MockRouter },
+                {
+                    provide: ActivatedRoute,
+                    useValue: {
+                        paramMap: of({
+                            get: (key: string) => {
+                                switch (key) {
+                                    case 'tutorialGroupId':
+                                        return 1;
+                                }
+                            },
+                        }),
+                        parent: {
+                            parent: {
+                                paramMap: of({
+                                    get: (key: string) => {
+                                        switch (key) {
+                                            case 'courseId':
+                                                return 1;
+                                        }
+                                    },
+                                }),
+                            },
+                        },
+                    },
+                },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                editTutorialGroupComponentFixture = TestBed.createComponent(EditTutorialGroupComponent);
+                editTutorialGroupComponent = editTutorialGroupComponentFixture.componentInstance;
+            });
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it('should initialize', () => {
+        editTutorialGroupComponentFixture.detectChanges();
+        expect(editTutorialGroupComponent).not.toBeNull();
+    });
+
+    it('should set form data correctly', () => {
+        const tutorialGroupService = TestBed.inject(TutorialGroupsService);
+
+        const tutorialGroupOfResponse = new TutorialGroup();
+        tutorialGroupOfResponse.id = 1;
+        tutorialGroupOfResponse.title = 'test';
+
+        const response: HttpResponse<TutorialGroup> = new HttpResponse({
+            body: tutorialGroupOfResponse,
+            status: 200,
+        });
+
+        const findByIdStub = jest.spyOn(tutorialGroupService, 'getOneOfCourse').mockReturnValue(of(response));
+        const tutorialGroupFormStubComponent: TutorialGroupFormStubComponent = editTutorialGroupComponentFixture.debugElement.query(
+            By.directive(TutorialGroupFormStubComponent),
+        ).componentInstance;
+
+        editTutorialGroupComponentFixture.detectChanges();
+        expect(editTutorialGroupComponent.tutorialGroup).toEqual(tutorialGroupOfResponse);
+        expect(findByIdStub).toHaveBeenCalledWith(1, 1);
```

Comment:
Missing `toHaveBeenCalledOnce()`

Reply:
Done in d9aae13fd3c1aa34fdf4330d8e95e9c445163825